Use react-overlays on ActionMenu (#558)
vtex/styleguide: CHANGELOG.md

## [Unreleased]

### Changed

> - **ActionMenu** now uses `react-overlays` and is being added to the DOM in

Reviewer: this is an implementation detail that is not supposed to be relevant for the user; if this fixes an issue, it should be under `### Fixed` and refer only to the issue at hand

## [Unreleased]

### Fixed

> - **ActionMenu** unwanted behavior that made the component not visible when inside other components that use `overflow`.

Reviewer: ```suggestion
- **ActionMenu** unwanted behavior, where the menu would be cropped when inside other components that use `overflow: hidden`.
```
